### ISSUE TITLE
SingleTagField causes maximum recursion depth exceeded

### ISSUE THREAD

**Issue by panpanick** [radiac/django-tagulous#65]
If I have a SingleTagField on model, like this
```
engine = SingleTagField()
```
I get this error when attempting to create an object with that model
```
File "/Users/jeanne/Env/phobeus_env/lib/python3.7/site-packages/tagulous/models/descriptors.py", line 133, in get_manager
    manager = SingleTagManager(self, instance)
  File "/Users/jeanne/Env/phobeus_env/lib/python3.7/site-packages/tagulous/models/managers.py", line 49, in __init__
    self.tag_cache = self.get_actual()
  File "/Users/jeanne/Env/phobeus_env/lib/python3.7/site-packages/tagulous/models/managers.py", line 101, in get_actual
    self.flush_cache()
  File "/Users/jeanne/Env/phobeus_env/lib/python3.7/site-packages/tagulous/models/managers.py", line 64, in flush_cache
    if hasattr(self.instance, cache_name):
  File "/Users/jeanne/Env/phobeus_env/lib/python3.7/site-packages/tagulous/models/descriptors.py", line 143, in __get__
    manager = self.get_manager(instance, instance_type)
  File "/Users/jeanne/Env/phobeus_env/lib/python3.7/site-packages/tagulous/models/descriptors.py", line 133, in get_manager
    manager = SingleTagManager(self, instance)
  File "/Users/jeanne/Env/phobeus_env/lib/python3.7/site-packages/tagulous/models/managers.py", line 49, in __init__
    self.tag_cache = self.get_actual()
  File "/Users/jeanne/Env/phobeus_env/lib/python3.7/site-packages/tagulous/models/managers.py", line 101, in get_actual
    self.flush_cache()
  File "/Users/jeanne/Env/phobeus_env/lib/python3.7/site-packages/tagulous/models/managers.py", line 64, in flush_cache
    if hasattr(self.instance, cache_name):
  File "/Users/jeanne/Env/phobeus_env/lib/python3.7/site-packages/tagulous/models/descriptors.py", line 143, in __get__
    manager = self.get_manager(instance, instance_type)
  File "/Users/jeanne/Env/phobeus_env/lib/python3.7/site-packages/tagulous/models/descriptors.py", line 133, in get_manager
```

It repeats ad infinitum by nature of recursion, causing this error.

**Comment by radiac** [comment]
That's a bit odd - can you give me an example of how you're creating the object?

**Comment by panpanick** [comment]
I am creating the object in the Django admin panel. When entering the engine field, there is a blank dropdown where I can name the first tag. It is just like all of the other tag fields except that it is a dropdown.

The model looks something like this
```
class Game(models.Model):
  ...
  engine = SingleTagField()
  ...

```

Nothing else in the model is referencing or accessing the `engine` field.

**Comment by radiac** [comment]
This should be fixed in the latest version of Tagulous (0.14.0).
